client/my-sites/checkout/checkout-thank-you/domains/domain-thank-you.tsx

```diff
@@ -34,6 +34,10 @@ const DomainThankYou: React.FC< DomainThankYouContainerProps > = ( {
 	hideProfessionalEmailStep,
 	type,
 } ) => {
+	const {
+		data: { launchpad_screen: launchpadScreen, site_intent: siteIntent },
+	} = useLaunchpad( selectedSiteSlug );
+
 	const thankYouProps = useMemo< DomainThankYouProps >( () => {
```

Comment:
This seems like an unusual pattern each time I see it here - to write a 'props getter' method, and then wrap it useMemo. Makes it a bit hard to review because I don't understand the 'why' here.